README.md

```diff
@@ -1,5 +1,11 @@
 # hockey-gerrit
 
+[![Gem Version](https://badge.fury.io/rb/hockey-gerrit.svg)](https://badge.fury.io/rb/hockey-gerrit)
```

Comment:
It's better to link directly to the gem:
`https://rubygems.org/gems/hockey-gerrit`